src/plugins/liveobjects/liveobjectspool.ts

```diff
@@ -40,10 +45,123 @@ export class LiveObjectsPool {
     this._pool = this._getInitialPool();
   }
 
+  createZeroValueObjectIfNotExists(objectId: string): void {
+    if (this.get(objectId)) {
+      return;
+    }
+
+    const parsedObjectId = ObjectId.fromString(this._client, objectId);
+    let zeroValueObject: LiveObject;
+    switch (parsedObjectId.type) {
+      case 'map': {
+        zeroValueObject = new LiveMap(this._liveObjects, MapSemantics.LWW, null, objectId);
```

Comment:
would be nice to make static constructors for these so that the implementation of creating a zero value lives on the class
```suggestion
        zeroValueObject = LiveMap.ZeroValue();
```

Reply:
good idea, will do

Reply:
added in https://github.com/ably/ably-js/pull/1897/commits/3ed285d2d31a12ab6112cd366ef3e5ab5d864df9